rbac/management/audit_log/view.py

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""View for Audit Logs."""
+from management.models import AuditLog
+from management.serializers import AuditLogSerializer
+from rest_framework import mixins, viewsets
+from rest_framework.permissions import AllowAny
+
+
+class AuditLogViewSet(mixins.ListModelMixin, viewsets.GenericViewSet):
+    """Audit Logs View.
+
+    A viewset that provides default `list()` actions.
+
+    """
+
+    queryset = AuditLog.objects.all()
```

Comment:
What is this used for ? queries with .all() are suspicious. I think after you add tenancy - there should be at condition for tenant_id.